components/partials/Footer.vue

```diff
@@ -94,10 +95,12 @@ export default {
   computed: {
     currentTheme: {
       get () {
-        return this.themes.map(l => l.value).indexOf(this.$store.state.storage.theme)
+        const index = this.themes.map(l => l.value).indexOf(this.$theme.value)
```

Comment:
- What about `const index = this.themes.findIndex(l => l.value === this.$theme.value) `?
- Statically generated version focuses on the default selected item until the client-side is being loaded. Not a problem for us because the button is in footer out of view and also no hydration because the markup is the same just attr change happening. Just caution this problem is still there for general usage:

![image](https://user-images.githubusercontent.com/5158436/75708999-2dcc5c80-5cc2-11ea-9ab8-749e221f6ff3.png)

Reply:
Well `findIndex` has bad IE support:
<img width="1112" alt="Screenshot 2020-03-09 at 11 29 39" src="https://user-images.githubusercontent.com/904724/76204775-54762000-61f9-11ea-88bf-b086ceb25af0.png">

Reply:
It is already transpiled by babel :)